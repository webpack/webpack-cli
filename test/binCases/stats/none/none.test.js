"use strict";

const { run } = require("../../../testUtils");

test("none", () => {
	const { code, stdout, stderr } = run(__dirname);

	expect(code).toBe(0);
	expect(stdout).toHaveLength(0);
	expect(stderr).toHaveLength(0);
});
