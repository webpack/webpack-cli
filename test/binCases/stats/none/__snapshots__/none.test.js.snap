// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`none 1`] = `""`;
